### PR TITLE
Item upload date property, pagination

### PR DESCRIPTION
**Pull request by aauga**
What has been done:
- Added "uploaded" property to item;
- Item sort by date in descending order when making GET request;
- Created pagination. Item GET response will have two pairs pagination and items.

### CODE REVIEW

File: Application/Items/Commands/Create.cs [hunk 1]
```diff
@@ -60,18 +53,21 @@ public async Task<Unit> Handle(Command request, CancellationToken cancellationTo
                 }
 
                 var user = await _context.Users.FindAsync(request.User);
-                request.Item.Uploader = user.Username;
+                
+                // gonna leave this as temp until front is connected to back
+                request.Item.Uploader = "temp";
```
Comment (aauga) [inline Application/Items/Commands/Create.cs:58]: this is a temporary thing, until the front is connected to the back. will change back later

File: Application/Items/Queries/List.cs [hunk 2]
```diff
@@ -43,8 +47,14 @@ public async Task<IEnumerable<Item>> Handle(Query request, CancellationToken can
                     
                     item.ImageUrls = urls;
                 }
+
+                var totalItems = _context.Items.Count();
```
Comment (kristupas-g) [inline Application/Items/Queries/List.cs:51]: Would be great if you moved pagination logic to another function

File: Domain/Entities/Item.cs [hunk 3]
```diff
@@ -23,6 +23,6 @@ public class Item
         public List<String> ImageUrls { get; set; }
         public string Uploader { get; set; }
         public string Redeemer { get; set; }
-       
+        public DateTime Uploaded { get; set; }
```
Comment (kristupas-g) [inline Domain/Entities/Item.cs:26]: Why do we need Upload date?

File: WebApi/Controllers/ItemsController.cs [hunk 4]
```diff
@@ -14,16 +14,10 @@ namespace WebApi.Controllers
...
-            var items = await Mediator.Send(new List.Query());
-
-            if (!items.Any())
-            {
-                return NoContent();
-            }
-
-            return Ok(items);
+            var obj = await Mediator.Send(new List.Query {Page = page, ItemsPerPage = itemsPerPage});
+            return Ok(obj);
```
Comment (kristupas-g) [inline WebApi/Controllers/ItemsController.cs:20]: We should still check if MediatR query returned any items

**Pull request closed without merging**